frontend/pages/_app.tsx

```diff
@@ -22,6 +22,9 @@ import { LayoutStaticProp } from 'types';
 
 import 'styles/globals.css';
 
+require('dayjs/locale/es');
+require('dayjs/locale/pt');
```

Comment:
I think we could make the code more flexible if we use the locales defined in `locales.config.json`:
```suggestion
import localesConfig from 'locales.config.json';

// Enable dayjs to display translated dates in all the application's locales
localesConfig.locales.forEach(({ locale }) => require(`dayjs/locale/${locale}.js`));
```